src/emm/services/github_service.py

```diff
@@ -19,6 +20,10 @@ def __init__(self, gh_cli: local) -> None:
     @classmethod
     def create(cls) -> local:
         """Initialize the github cli in command line."""
+        if not shutil.which("gh"):
+            raise EnvironmentError(
+                "Please make sure you've installed github CLI. https://cli.github.com/"
+            )
```

Comment:
Error message would be like:
```

Traceback (most recent call last):
  File "/Users/siran/Library/Caches/pypoetry/virtualenvs/evg-module-manager-UX1Q_zI2-py3.9/bin/evg-module-manager", line 5, in <module>
    cli()
  File "/Users/siran/Library/Caches/pypoetry/virtualenvs/evg-module-manager-UX1Q_zI2-py3.9/lib/python3.9/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/Users/siran/Library/Caches/pypoetry/virtualenvs/evg-module-manager-UX1Q_zI2-py3.9/lib/python3.9/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/Users/siran/Library/Caches/pypoetry/virtualenvs/evg-module-manager-UX1Q_zI2-py3.9/lib/python3.9/site-packages/click/core.py", line 1656, in invoke
    super().invoke(ctx)
  File "/Users/siran/Library/Caches/pypoetry/virtualenvs/evg-module-manager-UX1Q_zI2-py3.9/lib/python3.9/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/siran/Library/Caches/pypoetry/virtualenvs/evg-module-manager-UX1Q_zI2-py3.9/lib/python3.9/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/Users/siran/Library/Caches/pypoetry/virtualenvs/evg-module-manager-UX1Q_zI2-py3.9/lib/python3.9/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/siran/Documents/evg-module-manager/src/emm/emm_cli.py", line 175, in cli
    inject.configure(dependencies)
  File "/Users/siran/Library/Caches/pypoetry/virtualenvs/evg-module-manager-UX1Q_zI2-py3.9/lib/python3.9/site-packages/inject/__init__.py", line 356, in configure
    _INJECTOR = Injector(config, bind_in_runtime=bind_in_runtime)
  File "/Users/siran/Library/Caches/pypoetry/virtualenvs/evg-module-manager-UX1Q_zI2-py3.9/lib/python3.9/site-packages/inject/__init__.py", line 202, in __init__
    config(binder)
  File "/Users/siran/Documents/evg-module-manager/src/emm/emm_cli.py", line 173, in dependencies
    binder.bind(GithubService, GithubService.create())
  File "/Users/siran/Documents/evg-module-manager/src/emm/services/github_service.py", line 24, in create
    raise EnvironmentError("Please make sure you've installed github CLI. https://cli.github.com/")
OSError: Please make sure you've installed github CLI. https://cli.github.com/
```

Reply:
I'm not sure if this is the behavior we want, we're still outputting a pretty huge traceback. Can we just output the error message?

Reply:
new error output:

`Please make sure you've installed github CLI. https://cli.github.com/`

Reply:
Maybe something like `Please make sure you've installed the Github CLI. Instructions on how to do so can be found at https://cli.github.com/`